### PR TITLE
More speed optimizations: Images again, fonts CSS

### PR DESCRIPTION
**Pull request by baltpeter**

### CODE REVIEW

**Comment by baltpeter** [comment]
This is now ready for review.

**Comment by mal-tee** [comment]
Sidenote: I think we could lazy load the itz-logo as well?

https://github.com/datenanfragen/website/blob/b_make-google-happy/layouts/_default/baseof.html#L175

**Comment by baltpeter** [comment]
> Side note: I think we could lazy load the itz-logo as well?

Sure.

**Review by mal-tee — DISMISSED** [review]
Wow, very nice! Thank you! What do you think: Is making tests for the "image pipeline" worth the effort?

**Comment by baltpeter** [comment]
> What do you think: Is making tests for the "image pipeline" is worth the effort?

What would you want to test in particular?

**Comment by mal-tee** [comment]
Alt texts, correct height/width-ratio, that an image is shown in the first place, not the scaling or format.

File: cypress/integration/components/hugo-images.spec.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            ]) {
+                const source = win.document.querySelector(`picture source:nth-of-type(${num})`);
+                expect(source.srcset).to.include(format);
+                expect(source.srcset).to.not.include(format === '.webp' ? '.jpg' : '.webp');
+            }
+        });
+    });
+
+    it('Featured images are not lazy-loaded', () => {
+        cy.get('picture img').should('not.have.attr', 'loading');
```
Comment (mal-tee) [inline cypress/integration/components/hugo-images.spec.js:60]:
```suggestion
        cy.get('picture img').first().should('not.have.attr', 'loading');
```

Seems cleaner to me.

**Review by mal-tee — APPROVED** [review]
Thank you!

**Pull request merged**